lib/reline/ansi.rb

```diff
@@ -64,18 +65,33 @@ def self.set_default_key_bindings(config, allow_terminfo: true)
     end
   end
 
+  def self.set_default_key_bindings_ansi_cursor(config)
+    ANSI_CURSOR_KEY_BINDINGS.each do |char, (default_func, modifiers)|
+      bindings = [["\e[#{char}", default_func]] # CSI + char
+      if modifiers[:ctrl]
+        # CSI + ctrl_key_modifier + char
+        bindings << ["\e[1;5#{char}", modifiers[:ctrl]]
```

Comment:
In https://mdfs.net/Docs/Comp/Comms/ANSIKeys, it says `"\e[1;5D"` and also `"\e[5D"` is valid as Ctrl+Left.
Although, I think we only need `1;5` style because `set_default_key_bindings_comprehensive_list` only had `\e[1;5D`, emacs and readline only supports `\e[1;5D` style.